docs/reference/announcements-release-notes/850/850-announcements.md

```diff
@@ -84,42 +63,26 @@ Until a fix is delivered in 8.5.1, workarounds include not deserializing timesta
 | :------------- | :----------------- |
 | 9 January 2024 | 9 July 2025        |
 
-:::caution
-The [form linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#using-the-link-button) feature is impacted by an [issue](https://github.com/camunda/camunda/issues/16311) where the wrong forms can get linked with new user task instances, effectively corrupting the user task instance. If you make use of this feature and run either `8.4.0`, `8.4.1` or `8.4.2`, we urge you to update to the newest `8.4.3` patch that includes the required fix.
-
-Follow the instructions in the [form linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#known-issues-with-linked-forms) documentation to resolve this issue.
-:::
+### Supported environment changes
 
-### Versioning changes in Helm chart
+#### Versioning changes in Helm chart
 
 As of the 8.4 release, the Camunda 8 **Helm chart** version is decoupled from the version of the application. The Helm chart release still follows the applications release cycle, but it has an independent version. (e.g., in the application release cycle 8.4, the chart version is 9.0.0).
 
 For more details about the applications version included in the Helm chart, review the [full version matrix](https://helm.camunda.io/camunda-platform/version-matrix/).
 
-### Dockerfile numeric ID
-
-The Dockerfile now uses a numeric user ID instead of a non-numeric user.
-This will allow the Helm users to use `runAsNonRoot=true` without the need to explicitly set the ID in the Helm `values.yaml` file.
-
-### Deprecated in 8.4
-
-The [Zeebe configuration properties for Camunda Identity](/self-managed/zeebe-deployment/configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity)
-were deprecated in `8.4`. Please use the dedicated Camunda Identity properties or the [corresponding environment variables](/self-managed/identity/deployment/configuration-variables.md#core-configuration).
-
-### Versioning changes in Elasticsearch
+#### Versioning changes in Elasticsearch
 
 As of the 8.4 release, Camunda is compatible with Elasticsearch 8.9+ and no longer supports older Elasticsearch versions. See [supported environments](/reference/supported-environments.md).
```

Comment:
Elasticsearch is noted in 8.3 and 8.4, but I do not see this in any other version.